mapcompose/src/main/java/ovh/plrapps/mapcompose/ui/view/TileCanvas.kt

```diff
@@ -27,7 +28,8 @@ internal fun TileCanvas(
     alphaTick: Float,
     colorFilterProvider: ColorFilterProvider?,
     tilesToRender: List<Tile>,
-    isBitmapFilteringEnabled: Boolean
+    bitmapFilteringPredicate: (MapState) -> Boolean,
+    mapState: MapState
```

Comment:
The `TileCanvas` doesn't need to depend on `MapState`. Change `bitmapFilteringPredicate` type to `() -> Boolean` and capture de `MapState` in a lambda passed by the parent.
Remove `mapState` parameter

Reply:
Also, rename `bitmapFilteringPredicate` to `isFilteringBitmap`.

Reply:
> The TileCanvas doesn't need to depend on MapState.

Which is why I originally wanted it to only required the scale, not the entire `MapState`, but that's a neat solution, done.